addon/models/node.js

```diff
@@ -40,52 +36,21 @@ export default OsfModel.extend({
         inverse: 'children'
     }),
     children: DS.hasMany('nodes', {
-        inverse: 'parent',
-        updateRequest: {
-            requestType: () => 'POST'
```

Comment:
For my own referebce: This relationship meta config field was required to make comments work. Will the removal affect the new PR as well?